.github/workflows/main.yml

```diff
@@ -16,6 +16,14 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Setup Go
```

Comment:
instead of setting up go and then installing nxt, you could just do 

`sudo snap install nxt-router --edge` and then just do `nxt init && nxt start`